pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```
@@ -160,12 +179,13 @@ public static boolean patch(@NonNull String id, @NonNull String user, @NonNull N
         return true;
     }
 
-    public static Boolean isValueUnique(@NonNull DBElement dbElement) {
+    public static Boolean isValueUnique(@NonNull DBElement dbElement, String value) {
```

This method needs some work since it appears (at least) exceptions are swallowed.